tests/test_optics.py

```diff
@@ -208,13 +255,26 @@ def test_disk():
         assert model.kwargs[key] == model2.kwargs[key], 'key %r mismatch'%key
     for key in model.opt_kwargs:
         assert key in model2.opt_kwargs, 'key %r missing from model2 opt_kwargs'%key
-        assert model.opt_kwargs[key] == model2.opt_kwargs[key], 'key %r mismatch'%key
-    for key in model.atm_kwargs:
-        assert key in model2.atm_kwargs, 'key %r missing from model2 atm_kwargs'%key
-        assert model.atm_kwargs[key] == model2.atm_kwargs[key], 'key %r mismatch'%key
-    assert model.sigma == model2.sigma,'sigma mismatch'
-    assert model.g1 == model2.g1,'g1 mismatch'
-    assert model.g2 == model2.g2,'g2 mismatch'
+        if type(model.opt_kwargs[key])==np.ndarray:
+            np.testing.assert_almost_equal(model.opt_kwargs[key],model2.opt_kwargs[key],err_msg='key %r mismatch' % key)
+        else:
+            assert model.opt_kwargs[key] == model2.opt_kwargs[key], 'key %r mismatch'%key
+    for key in model.gsparams_kwargs:
+        assert key in model2.gsparams_kwargs, 'key %r missing from model2 gsparams_kwargs'%key
+        assert model.gsparams_kwargs[key] == model2.gsparams_kwargs[key], 'key %r mismatch'%key
+    for key in model.other_kwargs:
+        assert key in model2.other_kwargs, 'key %r missing from model2 other_kwargs'%key
+        assert model.other_kwargs[key] == model2.other_kwargs[key], 'key %r mismatch'%key
+    assert model.atmo_type == model2.atmo_type,'atmo_type mismatch'
+
+@timer
+def test_makestars(nstars=100,constant_atmoparams=True,template='des'):
+
+    print("test_makestars, n=%d, constant_atmoparams=%d" % (nstars,constant_atmoparams))
+    model = piff.Optical(template=template,atmo_type='VonKarman',gsparams='starby2')
+    params = random_params(nstars,model,constant_atmoparams=constant_atmoparams)
+    stars = make_stars(nstars,model,params)
```

Comment:
What is the point of this test?  There are no asserts, and nothing is being done with the final `stars` variable, so I'm not sure what is actually being tested here.

Reply:
This is to test that the code runs in the mode used by optatmo in a reasonable amount of time, and to compare the timing with different options.   Is there a good way to test for speed, that would work on different machines?

Reply:
If that's all it's for, then it shouldn't be in the CI test suite.

Rather, you can put this bit in a `time_optics.py` file, which you could run on different machines to check how long it takes.  Probably living in the devel directory rather than here.  We have a number of that kind of thing in GalSim for instance, which have been very useful for various speed optimizations.

Reply:
moved to devel/time_optics.py